packages/docusaurus-plugin-content-docs/src/globalData.ts

```diff
@@ -24,6 +24,7 @@ function toGlobalDataDoc(doc: DocMetadata): GlobalDoc {
     id: doc.unversionedId,
     path: doc.permalink,
     sidebar: doc.sidebar,
+    unlisted: doc.unlisted,
```

Comment:
Do not add attribute if not unlisted => save global data space

(we could do the same for sidebar, but IMHO most docs are inside a sidebar so the impact is much smaller)